PT-BR Translation pt.po

mauricio-sonegatti opened the pull request:
Creation of pt-br translation, based on @joebordes translation for spanish.
The file had to be changed to pt.br because the pt-br.po file does not build.

mauricio-sonegatti committed:
PT-BR Translation pt.po

Creation of pt-br translation, based on @joebordes translation for spanish.
The file had to be changed to pt.br because the pt-br.po file does not build.

mazameli commented:
Wow! This is great, Mauricio. Just so you know, a few minutes ago we merged https://github.com/metabase/metabase/pull/7372 into master which externalizes a number of backend strings, plus a handful of front-end ones we missed.

fernandosjp commented on locales/pt.po:
```diff
@@ -0,0 +1,8573 @@
+
+#: frontend/src/metabase/admin/databases/components/DatabaseSchedulingForm.jsx:125
+msgid "Database syncing"
+msgstr "Sincronizando Banco de Dados"
```
I would keep database in lower case "banco de dados"

fernandosjp reviewed (commented):
@mauricio-sonegatti I didn't have time to read the full document but I noted some stuff would be worth looking at! If you need help let us know here!

mauricio-sonegatti committed:
Translation Update

Translated new backend strings, rechecked some old ones.

mauricio-sonegatti committed:
Translation Fixes

Some new fixes

dannyeuu commented on locales/pt.po:
```diff
@@ -0,0 +1,8887 @@
+
+#: frontend/src/metabase/admin/databases/containers/DatabaseEditApp.jsx:150
+msgid "Scheduling"
+msgstr "Programação"
```
I think @mauricio-sonegatti should be a better translation `Agendamento`

dannyeuu commented on locales/pt.po:
```diff
@@ -0,0 +1,8887 @@
+#: frontend/src/metabase/query_builder/components/AlertModals.jsx:336
+#: frontend/src/metabase/reference/components/RevisionMessageModal.jsx:47
+msgid "Save changes"
+msgstr "Salvar mudanças"
```
Here `mudanças` to `alterações`

dannyeuu commented on locales/pt.po:
```diff
@@ -0,0 +1,8887 @@
+#: frontend/src/metabase/admin/datamodel/containers/TableSettingsApp.jsx:112
+#: frontend/src/metabase/admin/datamodel/containers/TableSettingsApp.jsx:120
+msgid "Starting…"
+msgstr "Começando ..."
```
here `Iniciando`

dannyeuu commented on locales/pt.po:
```diff
@@ -0,0 +1,8887 @@
+
+#: frontend/src/metabase/admin/datamodel/components/ObjectActionSelect.jsx:59
+msgid "Revision History"
+msgstr "Rever História"
```
Maybe can be better with `Rever Histórico`

dannyeuu commented on locales/pt.po:
```diff
@@ -0,0 +1,8887 @@
+#: frontend/src/metabase/admin/datamodel/components/PartialQueryBuilder.jsx:118
+#: frontend/src/metabase/public/components/widgets/AdvancedEmbedPane.jsx:110
+msgid "Preview"
+msgstr "Vista preliminar"
```
Maybe `Visualizar`

dannyeuu reviewed (commented):
It should be reviewed this translation concordance in the Portuguese language, looks like translated by Google Translator.

macvirii committed:
String updates and corrections

Pushed the new strings as 2018-06-28, some translations might be wrong, the update-translations command made some weird alterations. Some errors could have passed by when I was reviewing.

mauricio-sonegatti commented:
As stated on the 1a01c68 commit... I updated the translations, some weird behavior happened in update-translations. Something that wasn't marked fuzzy might escaped me.
For now the translation seems adequate, but I can't test it.
After building the latest version, the translation isn't showing. Even selecting it with the admin option it doesn't do anything. 
I'm gonna keep trying.

kdoh commented:
Going to go ahead and close this to reflect our switch to POEditor for translations. [Here's the PR-BR page](https://poeditor.com/projects/po_edit?id_language=190&id=200535) there in case anyone needs it.
